maths/arc_length.py

```diff
@@ -7,6 +7,8 @@ def arc_length(angle: int, radius: int) -> float:
     3.9269908169872414
     >>> arc_length(120, 15)
     31.415926535897928
+    >>> arc_length(90, 10)
+    15.7079632679
```

Comment:
```suggestion
    >>> arc_length(90, 10)
    15.707963267948966
```
You just need to update the expected value; the only reason why this test is failing is because the function has a higher decimal precision than your provided value:
```
010     >>> arc_length(90, 10)
Expected:
    15.7079632679
Got:
    15.707963267948966
```